spec/features/workflow/drafting_requirements_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+RSpec.feature "Drafting requirements" do
+  scenario do
+    given_there_is_a_document_with_no_content
+    when_i_visit_the_edit_document_page
+    and_save_without_inputting_anything
+    then_i_should_see_an_error_message_regarding_a_missing_title
+
+    when_i_visit_the_edit_document_page
+    and_save_a_title_which_doesnt_meet_the_validations
+    then_i_should_see_error_messages_regarding_validations
+
+    when_i_visit_the_edit_document_page
```

Comment:
We don't need this one as it's covered by other features :-)

Reply:
Good spot